Projekty/Projekt1/AdriannaGrudzień_KingaUłasik/WUM_projekt1_inżynieria_cech.ipynb

```diff
@@ -0,0 +1,1810 @@
+{
```

Comment:
<p>get_dummies() ma parametr drop_first - usuń pierwszą kolumnę. Domyślnie jest na False. Jeśli jej nie usuniemy, to mamy idealną współzależność 1 kolumn od innych</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>